90-aws-kinesis-customized-event-source/aws-kinesis-source.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Apache Camel AWS Kinesis Source
+#
+apiVersion: sources.knative.dev/v1alpha1
+kind: CamelSource
+metadata:
+  name: camel-aws-kinesis-source
+spec:
+  source:
+    integration:
+      configuration:
+        # What other types do we have?
+        - type: secret
+          value: aws-kinesis
+        - type: env
+          value: AWS_HOST=localstack:4568
+        - type: property
+          value: camel.component.aws-kinesis.configuration=#class:org.apache.camel.k.examples.CustomKinesisConfiguration
+      dependencies:
+        # Needed for the json part on the flow/steps below
+        - camel:jackson
+        # Needed for the component
+        - camel:camel-aws-kinesis
+        # Provides the custom Kinesis configuration
+        - mvn:org.apache.camel.k.examples/custom-kinesis-configuration:1.0.1
```

Comment:
Here you need to add also 
        - mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-cbor/2.10.4
Otherwise it will fail.

Reply:
In this case it is not needed because the custom client is specifically disabling cbor. 

```System.setProperty("com.amazonaws.sdk.disableCbor", "true");```

Maybe I should add a comment about this?

Reply:
On a second thought: this can be confusing. I'll add more details about it.

Reply:
I think a comment is good in this case.